producer/selector_test.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 func TestRoundRobin(t *testing.T) {
-	queues := make([]*primitive.MessageQueue, 10)
```

Comment:
why 10-> 0 @cserwen

Reply:
make() will init the slice 
![image](https://user-images.githubusercontent.com/46882838/234830945-c6b34510-65ec-490b-aca8-82794f9fceb3.png)